### ISSUE TITLE
Validation Error in Plugin

### ISSUE THREAD

**Issue by spg-matt** [samtecspg/conveyor#147]
When creating a channel from a source the validation on the parameters can fail, but the page acts as if the channel was created. To reproduce create a source template channel. If you fail to give it a name or description the validation **works**. However if you fail to give it any other required parameter you get an error and success message. This is what is in the browser console:

```
[Log] Error (conveyor.bundle.js, line 27)
Object

error: "Bad Request"

message: "The following parameters didn't pass validation:↵[password = undefined] => Required.↵[singleList = [[object Object]]] => Invalid value(s)…"

statusCode: 400

Object Prototype
```

and in the terminal

```
api_1            | Error: The following parameters didn't pass validation:
api_1            | [password = undefined] => Required.
api_1            | [singleList = [[object Object]]] => Invalid value(s). Available values [1,2,4].
api_1            | [multiList = [[object Object],[object Object]]] => Invalid value(s). Available values [1,2,4].
api_1            | [advText1 = null] => Required.
api_1            |     at Function.save (/usr/src/app/models/flow.model.js:126:27)
api_1            |     at FlowTemplate.findByName (/usr/src/app/modules/flow/controllers/add.flow.controller.js:30:21)
api_1            |     at ES.searchByQuery (/usr/src/app/models/flow-template.model.js:216:20)
api_1            |     at Client.search (/usr/src/app/datasources/elasticsearch.ds.js:161:20)
api_1            |     at bound (domain.js:396:14)
api_1            |     at runBound (domain.js:409:12)
api_1            |     at respond (/usr/src/app/node_modules/elasticsearch/src/lib/transport.js:328:9)
api_1            |     at checkRespForFailure (/usr/src/app/node_modules/elasticsearch/src/lib/transport.js:266:7)
api_1            |     at HttpConnector.<anonymous> (/usr/src/app/node_modules/elasticsearch/src/lib/connectors/http.js:159:7)
api_1            |     at IncomingMessage.bound (/usr/src/app/node_modules/elasticsearch/node_modules/lodash/dist/lodash.js:729:21)
api_1            |     at IncomingMessage.emit (events.js:187:15)
api_1            |     at IncomingMessage.EventEmitter.emit (domain.js:460:23)
api_1            |     at endReadableNT (_stream_readable.js:1081:12)
api_1            |     at process._tickCallback (internal/process/next_tick.js:63:19)
```

Good thing that it is erroring, but the error isn't being handled. As the page acts like the channel was saved:

![image](https://user-images.githubusercontent.com/35550124/43851920-a75deec8-9ad7-11e8-9971-4542c733bd53.png)

**Comment by malave** [comment]
@spg-matt please test and let me know if the issue persists.

**Comment by wrathagom** [comment]
Closed by f061a3d71fe88daa6ddc632e6e58ddd711f9579d
